packages/runtime/src/useCases/transport/messages/SendMessage.ts

```diff
@@ -73,21 +74,53 @@ export class SendMessageUseCase extends UseCase<SendMessageRequest, MessageDTO>
             );
         }
 
-        if (!(transformedContent instanceof Request)) return;
+        if (transformedContent instanceof Notification) return;
+        if (!(transformedContent instanceof Notification)) {
+            const peerDeletedAddressArray: string[] = [];
+            const peerToBeDeletedAddressArray: string[] = [];
+            const missingOrInactiveRelationshipAddressArray: string[] = [];
+            const recipientsCoreAddress = recipients.map((r) => CoreAddress.from(r));
```

Comment:
Why do you transform the strings to CoreAddresses and then later only use the strings again?

Reply:
I have changed it like @britsta suggested.